packages/adapters-library/src/scripts/helpers.ts

```diff
@@ -55,9 +55,11 @@ export class Helpers {
       (token) => token.address === protocolTokenAddress,
     )
     if (!protocolToken) {
-      throw new Error(
-        `Protocol token with address ${protocolTokenAddress} not found`,
-      )
+      // Ideally we want to throw here but when filterProtocolToken is included in defiProvider requests it will throw which makes the snapshots include many errors which is confusing
+      // We need to improve the flows when filterProtocolToken is included in the request
+      // JP to discuss with Bernardo
+
+      return {} as ProtocolToken<AdditionalMetadata>
```

Comment:
It's not a blocker, but it's definitely a priority to get this sorted.

Reply:
Where is this happening? I have tried re-creating multiple snapshots and I didn't encounter any issues.